### PR TITLE
Add MIRS sea-ice product to ioda converter

### PR DESCRIPTION
**Pull request by apchoiCMD**
#### Description
- Task for adding a new ioda converter for new sea-ice concentration
- For sea-ice, AMSR2 and MIRS product will be used

Partially addressed #1182

### CODE REVIEW

**Review by guillaumevernieres — COMMENTED** [review]
Looks good but the test cdl files need to be smaller. 15,000 lines is definitely too large, target something ~1000 lines or less.

File: utils/obsproc/IcecMirs2Ioda.h [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        std::stringstream ss;
+        ss << std::setfill('0')
+           << std::setw(4) << year << '-'
+           << std::setw(2) << month << '-'
+           << std::setw(2) << day << 'T'
+           << std::setw(2) << hour << ':'
+           << std::setw(2) << minute << ':'
+           << std::setw(2) << second << 'Z';
+        std::string formattedDateTime = ss.str();
+        util::DateTime dateTime(formattedDateTime);
```
Comment (AndrewEichmann-NOAA) [inline utils/obsproc/IcecMirs2Ioda.h:127]: I wonder if there's a utility for this

**Comment by AndrewEichmann-NOAA** [comment]
Looks reasonable

File: utils/obsproc/IcecMirs2Ioda.h (hunk 1 above)
Comment (apchoiCMD) [reply utils/obsproc/IcecMirs2Ioda.h:127]: IODA util has it- We have used it to another converters, too

**Comment by apchoiCMD** [comment]
> Looks good but the test cdl files need to be smaller. 15,000 lines is definitely too large, target something ~1000 lines or less.

Replaced with sub-sampled ones, now ~960 lines-

**Review by guillaumevernieres — APPROVED** [review]

**Pull request merged**